exercises/practice/sum-of-multiples/.approaches/from-factors/content.md

```diff
@@ -0,0 +1,103 @@
+# Calculating sum from factors
+
+```rust
+use std::collections::BTreeSet;
```

Comment:
Good question. I know I started with a `HashSet` in my first solution, but I don't remember _exactly_ why I switched. Nevertheless, when I run my benchmark on my machine, using a `HashSet` instead of a `BTreeSet` is about 2x slower.

This is a bit moot anyway since sets are going away. 🙂